zsh/.zshrc

```diff
@@ -257,6 +272,17 @@ if [[ -z "$LS_COLORS" ]]; then
   export LS_COLORS='di=1;34;40:ln=35;40:so=32;40:pi=33;40:ex=31;40:bd=34;46:cd=34;43:su=0;41:sg=0;46:tw=0;42:ow=0;43:'
 fi
 
+# Should this be inside load-our-ssh-keys()?
+if ! can_haz ssh-askpass; then
+  echo
+  echo "For the quickstart to prompt for your ssh key passphrase on the command line,"
+  echo "enable the following zqs setting: "
+  echo $(tput setaf 2) "zqs enable-ssh-askpass-require true" $(tput setaf 7)
```

Comment:
I was testing this here to see if we can make it stand out on the terminal. It will set the echo command to be green text and then back to white. I can remove if you don't think it makes sense.

Reply:
I think there's a `tput` that does a color reset instead of white. The user's terminal may be configured to have colored text.

Reply:
If we can do that reset, I definitely like making it stand out.

Reply:
I will do some more searching for that and test it works.

Reply:
I have found and tested the `tput setaf` command that resets back to the terminals default colours.